flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/FlinkSource.java

```diff
@@ -185,7 +186,7 @@ public Builder exposeLocality(boolean newExposeLocality) {
     }
 
     public Builder nameMapping(String nameMapping) {
-      readOptions.put(DEFAULT_NAME_MAPPING, nameMapping);
+      readOptions.put(TableProperties.DEFAULT_NAME_MAPPING, nameMapping);
```

Comment:
this is to revert the unintentional change from last PR, right?

Reply:
yes.